### PR TITLE
Retail use case with ECLASS semantics

### PR DESCRIPTION
**Pull request by sebastiankb**
Provided from @Christian-Block

### CODE REVIEW

**Comment by sebastiankb** [comment]
@ashimura I like to add @dezell as reviewer. Please can you add him. Thanks a lot.

File: USE-CASES/use-case-vertical-eclass-retail.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<section id="use-case-template">
+<h2>Retail</h2>
```
Comment (mlagally) [inline USE-CASES/use-case-vertical-eclass-retail.html:2]: Consider to add ECLASS to the title?

Comment (mlagally) [reply USE-CASES/use-case-vertical-eclass-retail.html:2]: Category should be Retail, title should be more specific.

Comment (mlagally) [reply USE-CASES/use-case-vertical-eclass-retail.html:2]: Proposal for a title:
Retail inventory and equipment management

File: USE-CASES/use-case-vertical-eclass-retail.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+</dd>
+<dt>Gaps</dt>
+<dd>
+There is no standardized semantic and vocabulary for product meta and master data as well as the InteractionAffordance.
+
+</dd>
+<dt>Existing standards</dt>
+<dd>
+
```
Comment (mlagally) [inline USE-CASES/use-case-vertical-eclass-retail.html:129]: Other standards should be mentioned as well :
- GS1
- W3C data set exchange
- 
Mapping of classifications across description schemes

File: USE-CASES/use-case-vertical-eclass-retail.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+<li>service provider</li>
+<li>device manufacturer</li>
+<li>gateway manufacturer</li>
+<li>network operator (potentially transparent for WoT use cases)</li>
+</ul>
+
+</dd>
+<dt>Motivation</dt>
+<dd>
+Every product has a supplier and buyer. To bring the two together, both the product description and the requirements must be exchanged. Ideally, these should be transferred in a standardized way to enable mapping as well as search and find.<br>
```
[the remark this answers is not in the document]
Comment (sebastiankb) [reply USE-CASES/use-case-vertical-eclass-retail.html:57]: This text block does not mention ECLASS. Below, ECLASS is shortly introduced and a reference is made to the webpage of ECLASS.

File: USE-CASES/use-case-vertical-eclass-retail.html [hunk 4]
```diff
@@ -126,14 +127,18 @@ <h2>Retail</h2>
 </dd>
 <dt>Existing standards</dt>
 <dd>
-
 The ECLASS Standard contains tens of thousands of product classes and unique properties. This lets you standardize procurement, storage, production, and distribution activities in and between companies - across sectors, countries and languages. Realize sales potentials and new synergies, reduce costs, and increase the efficiency of your merchandise and data management. <a href="https://www.eclass.eu/en">Get moren information about the ECLASS Standard on the website eclass.eu .</a>
-
 </dd>
 <dt>Comments</dt>
 <dd>
 The ECLASS association manages the ECLASS Standard and is currently working on an RDF representation of the dictionary to use it in WoT Thig Description.
```
[the remark this answers is not in the document]
Comment (mlagally) [reply USE-CASES/use-case-vertical-eclass-retail.html:134]: @sebastiankb , @Christian-Block 
Do you plan to provide additional text?

File: USE-CASES/use-case-vertical-eclass-retail.html [hunk 5]
```diff
@@ -126,14 +127,18 @@ <h2>Retail</h2>
...
-
 </dd>
-</dl>
+<dt>Other standards</dt>
+<dd>
+GS1
```
[the remark this answers is not in the document]
Comment (mlagally) [reply USE-CASES/use-case-vertical-eclass-retail.html:138]: @Christian-Block , @sebastiankb , can you please add links so we can merge this MR?

**Comment by mlagally** [comment]
Need to add references

**Pull request merged**